client/src/pages/Login/login.js

```diff
@@ -1,122 +1,122 @@
-import React, { useState } from "react";
-import { Grid, Typography, Paper, TextField, Button, InputAdornment } from "@material-ui/core";
-import AccountCircleIcon from '@material-ui/icons/AccountCircle';
-import LockIcon from '@material-ui/icons/Lock';
-import { useSelector, useDispatch } from "react-redux";
-import axios from "axios";
-import { useHistory } from "react-router-dom";
-import { useStyles } from "./styles";
-import { BACKEND_API } from "../../config/config";
-import { toast } from "react-toastify";
-import { loginSuccess } from "../../actions/user";
-import logo from '../../assets/images/Aaveg Glyph - Black.png';
-import bgimg  from '../../assets/images/loginPage.png'
+import React, { useState } from 'react'
+import { Grid, Typography, Paper, TextField, Button, InputAdornment } from '@material-ui/core'
+import AccountCircleIcon from '@material-ui/icons/AccountCircle'
+import LockIcon from '@material-ui/icons/Lock'
+import { useSelector, useDispatch } from 'react-redux'
+import axios from 'axios'
+import { useHistory } from 'react-router-dom'
+import { useStyles } from './styles'
+import { BACKEND_API } from '../../config/config'
+import { toast } from 'react-toastify'
+import { loginSuccess } from '../../actions/user'
+import logo from '../../assets/images/Aaveg Glyph - Black.png'
+import bgimg from '../../assets/images/loginPage.png'
 
 export const Login = (props) => {
-  const history = useHistory();
-  const classes = useStyles();
-  const dispatch = useDispatch();
+  const history = useHistory()
+  const classes = useStyles()
+  const dispatch = useDispatch()
 
-  const [webmail, setWebmail] = useState("");
-  const [password, setPassword] = useState("");
+  const [webmail, setWebmail] = useState('')
+  const [password, setPassword] = useState('')
 
-  const user = useSelector((state) => state.user);
+  const user = useSelector((state) => state.user)
 
   if (user !== null) {
-    history.push("/");
+    history.push('/')
   }
 
   const handleLogin = () => {
     const loginData = {
       email: webmail,
-      password: password,
-    };
+      password: password
+    }
 
     axios
-      .post(BACKEND_API + "/auth/login", JSON.stringify(loginData))
+      .post(BACKEND_API + '/auth/login', JSON.stringify(loginData))
       .then((res) => {
         if (res.status_code === 200) {
-          dispatch(loginSuccess(res.message));
-          toast.success("logged in/ get from res", {
-            position: "bottom-center",
-          });
-          history.push("/quiz");
+          dispatch(loginSuccess(res.message))
+          toast.success('logged in/ get from res', {
+            position: 'bottom-center'
+          })
+          history.push('/quiz')
         } else {
-          toast.error("error/get from res", { position: "bottom-center" });
+          toast.error('error/get from res', { position: 'bottom-center' })
```

Comment:
The same goes for error messages.
Toast styles are repeated. Define it once, use it many times.